parodos-model-api/src/main/java/com/redhat/parodos/workflow/task/WorkFlowTask.java

```diff
@@ -52,4 +58,26 @@ default List<WorkFlowTaskOutput> getWorkFlowTaskOutputs() {
 
 	}
 
+	default HashMap<String, Map<String, String>> getAsJsonSchema() {
+		HashMap<String, Map<String, String>> result = new HashMap<>();
+		taskParamsLoop: for (WorkFlowTaskParameter workFlowTaskParameter : this.getWorkFlowTaskParameters()) {
+			if (workFlowTaskParameter == null) {
+				continue;
+			}
+
+			if (workFlowTaskParameter.getType() == null) {
+				continue;
+			}
+
+			Map<String, String> properties = workFlowTaskParameter.getType().getAsJsonSchema();
+			properties.put("required", String.format("%s", !workFlowTaskParameter.isOptional()));
+			properties.put("description", workFlowTaskParameter.getDescription());
+			if (workFlowTaskParameter.getJsonSchemaOptions() != null) {
+				workFlowTaskParameter.getJsonSchemaOptions().forEach(properties::put);
```

Comment:
can you check if it's possible to simplify this in `getJsonSchemaOptions().putAll(properties);`?